jaeger-core/README.md

```diff
@@ -0,0 +1,11 @@
+# Core #
+If you only want to do custom instrumentation using the core tracing functionality then add the
+following:
+```xml
+<dependency>
+    <groupId>com.uber.jaeger</groupId>
+    <artifactId>jaeger-core</artifactId>
+    <version>$jaegerVersion</version>
+</dependency>
+```
+
```

Comment:
but overall I don't like that this is hidden, I think it belongs to the top level readme, since it's the core of this library, while the other modules are add-ons.